src/main/java/Duke.java

```diff
@@ -1,10 +1,74 @@
+package duke;
+
+import duke.Task;
+import duke.TaskType;
+import duke.Ui;
+import duke.TaskList;
+import duke.Storage;
+import duke.DukeException;
+import duke.Command;
+import duke.Parser;
+
+import java.io.FileNotFoundException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.time.format.FormatStyle;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Optional;
+import java.util.Scanner;
+
+/**
+ * Class contains main method of the Duke application.
+ * Duke manages, stores and track tasks as specified by the user.
+ */
 public class Duke {
+    private static String TASKS_PATHNAME = "data/tasks.txt";
+    private Ui ui;
+    private TaskList taskList;
+    private Storage storage;
+
+    /**
+     * main method of Duke
+     * @param args
+     */
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        new Duke(TASKS_PATHNAME).run();
     }
-}
+
+    /**
+     * Constructor which takes in file path of the storage file.
+     * @param filePath
```

Comment:
Perhaps add a description to the parameter name in the Javadoc comments to make it clearer?